main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        data = json.load(file)
+        for player in data:
+            if Player.objects.filter(nickname=player).exists():
+                continue
+            guild = data[player]["guild"]
+            race = data[player]["race"]
+            if guild is not None:
+                if not Guild.objects.filter(name=guild["name"]).exists():
+                    Guild.objects.create(
+                        name=guild["name"],
+                        description=guild["description"],
+                    )
+                guild = Guild.objects.filter(name=guild["name"]).get()
```

Comment:
Use get_or_create
https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create
Fix in all places